lib/ruby_lsp/ruby_lsp_rails/document_symbol.rb

```diff
@@ -93,25 +93,29 @@ def on_call_node_enter(node)
           )
         end
 
-        extract_callbacks(node)
-      end
-
-      private
-
-      sig { params(node: Prism::CallNode).void }
-      def extract_callbacks(node)
         receiver = node.receiver
         return if receiver && !receiver.is_a?(Prism::SelfNode)
 
-        message_value = node.message
+        message = node.message
```

Comment:
```suggestion
        message = T.must(node.message)
```
then we can remove the `T.must`s below.

Reply:
Not sure about this one.

`node.message` is nilable. Once we enter the switch-case, we know for sure that it's not `nil`, but I don't think we can guarantee that before entering.

Reply:
Merged it without this, but we can fix this in the next PR based on your thoughts.